pages/how-to-support/donate.js

```diff
@@ -47,6 +50,10 @@ const Donate = ({
       <CenteredText source={introMarkdown} />
     </PageSection>
 
+    <FormContainer>
+      <Markdown source={intro2Markdown} />
+    </FormContainer>
```

Comment:
The contenst of this `FormContainer` is not a form at all. Should we consider renaming the `FormContainer`, or possibly going for a different solution here?